class.jetpack-gutenberg.php

```diff
@@ -89,6 +94,47 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a block is able to run with current wp or gutenburg plugin versions
+	 *
+	 * @param array  $version_requirements An array containing wp and plugin version requirements.
+	 * @param string $extension_name The name of the block/plugin that has the version requirement.
+	 *
+	 * @since 8.4.0
+	 *
+	 * @return boolean True if the version of gutenberg required by the plugin is available.
+	 */
+	public static function is_editor_version_available( $version_requirements, $extension_name ) {
+		global $wp_version;
+
+		if ( defined( 'GUTENBERG_DEVELOPMENT_MODE' ) && GUTENBERG_DEVELOPMENT_MODE ) {
```

Comment:
It might be good to add a note about where this is set and why.

Reply:
Done